### PR TITLE
Update install-layout.asciidoc to reflect logging changes introduced in 8.6.0

### PR DESCRIPTION
**Pull request by rseldner**
Updated widget to reflect the logging changes introduced in 8.6.0 (v2 architecture) to close this issue:  https://github.com/elastic/ingest-docs/issues/175
1. switched agent log file from `elastic-agent-json.log` to `elastic-agent.ndjson`
2. with the exception of Endpoint, beats/component logs are written to `elastic-agent.ndjson` rather than their own log file.

I believe the logging changes were introduces in the v2 architecture:
https://github.com/elastic/elastic-agent/pull/1701

### CODE REVIEW

**Comment by rseldner** [comment]
I think this should be backported as far as 8.6 but not seeing a label for that version.

File: docs/en/ingest-management/tab-widgets/install-layout.asciidoc [hunk 1]
```diff
@@ -7,10 +7,8 @@
 Main {agent} configuration
 `/Library/Elastic/Agent/fleet.enc`::
 Main {agent} {fleet} encrypted configuration
-`/Library/Elastic/Agent/data/elastic-agent-*/logs/elastic-agent-json.log`::
-Log files for {agent}footnote:lognumbering[Logs file names end with a date and optional number: log-date.ndjson, log-date-1.ndjson, and so on as new files are created during rotation.]
-`/Library/Elastic/Agent/data/elastic-agent-*/logs/default/*-json.log`::
-Log files for {beats} shippersfootnote:lognumbering[]
+`/Library/Elastic/Agent/data/elastic-agent-*/logs/elastic-agent.ndjson`::
+Log files for {agent} and beats shippers footnote:lognumbering[Logs file names end with a date (`YYYYMMDD`) and optional number: `elastic-agent-YYYYMMDD.ndjson`, `elastic-agent-YYYYMMDD-1.ndjson`, and so on as new files are created during rotation.]
```
Comment (kilfoyle) [inline docs/en/ingest-management/tab-widgets/install-layout.asciidoc:11]:
```suggestion
Log files for {agent} and {beats} shippers footnote:lognumbering[Logs file names end with a date (`YYYYMMDD`) and optional number: `elastic-agent-YYYYMMDD.ndjson`, `elastic-agent-YYYYMMDD-1.ndjson`, and so on as new files are created during rotation.]
```

Just adding the attribute for "Beats" that's defined over [here](https://github.com/elastic/docs/blob/7470e3516e5298e2c540bd22f3281314c7e863e2/shared/attributes.asciidoc#L287).

File: docs/en/ingest-management/tab-widgets/install-layout.asciidoc [hunk 2]
```diff
@@ -26,10 +24,8 @@ command, use the `--base-path` CLI option to specify the custom base path.
 Main {agent} configuration
 `/opt/Elastic/Agent/fleet.enc`::
 Main {agent} {fleet} encrypted configuration
-`/opt/Elastic/Agent/data/elastic-agent-*/logs/elastic-agent-json.log`::
-Log files for {agent}footnote:lognumbering[]
-`/opt/Elastic/Agent/data/elastic-agent-*/logs/default/*-json.log`::
-Log files for {beats} shippers
+`/opt/Elastic/Agent/data/elastic-agent-*/logs/elastic-agent.ndjson`::
+Log files for {agent} and beats shippers footnote:lognumbering[]
```
Comment (kilfoyle) [inline docs/en/ingest-management/tab-widgets/install-layout.asciidoc:28]:
```suggestion
Log files for {agent} and {beats} shippers footnote:lognumbering[]
```

File: docs/en/ingest-management/tab-widgets/install-layout.asciidoc [hunk 3]
```diff
@@ -79,10 +71,8 @@ Shell wrapper installed into PATH
 Main {agent} configuration
 `/etc/elastic-agent/fleet.enc`::
 Main {agent} {fleet} encrypted configuration
-`/var/lib/elastic-agent/data/elastic-agent-*/logs/elastic-agent-json.log`::
-Log files for {agent}footnote:lognumbering[]
-`/var/lib/elastic-agent/data/elastic-agent-*/logs/default/*-json.log*`::
-Log files for {beats} shippers
+`/var/lib/elastic-agent/data/elastic-agent-*/logs/elastic-agent.ndjson`::
+Log files for {agent} and beats shippers footnote:lognumbering[]
```
Comment (kilfoyle) [inline docs/en/ingest-management/tab-widgets/install-layout.asciidoc:75]:
```suggestion
Log files for {agent} and {beats} shippers footnote:lognumbering[]
```

File: docs/en/ingest-management/tab-widgets/install-layout.asciidoc [hunk 4]
```diff
@@ -62,10 +56,8 @@ command, use the `--base-path` CLI option to specify the custom base path.
...
```
Comment (kilfoyle) [inline docs/en/ingest-management/tab-widgets/install-layout.asciidoc:60]:
```suggestion
Log files for {agent} and {beats} shippers footnote:lognumbering[]
```

File: docs/en/ingest-management/tab-widgets/install-layout.asciidoc [hunk 5]
```diff
@@ -45,10 +41,8 @@ command, use the `--base-path` CLI option to specify the custom base path.
 Main {agent} configuration
 `C:\Program Files\Elastic\Agent\fleet.enc`::
 Main {agent} {fleet} encrypted configuration
-`C:\Program Files\Elastic\Agent\data\elastic-agent-*\logs\elastic-agent-json.log`::
-Log files for {agent}footnote:lognumbering[]
-`C:\Program Files\Elastic\Agent\data\elastic-agent-*\logs\default\*-json.log*`::
-Log files for {beats} shippers
+`C:\Program Files\Elastic\Agent\data\elastic-agent-*\logs\elastic-agent.ndjson`::
+Log files for {agent} and beats shippers footnote:lognumbering[]
```
Comment (kilfoyle) [inline docs/en/ingest-management/tab-widgets/install-layout.asciidoc:45]:
```suggestion
Log files for {agent} and {beats} shippers footnote:lognumbering[]
```

**Review by kilfoyle — APPROVED** [review]
LGTM! Just a few minor suggestions to use the "Beats" attribute/variable.

As for backporting, our general policy is to backport only into active releases (the ones shown in the docs release dropdown).

Screenshot 2023-10-30 at 4 50 26 PM

**Comment by rseldner** [comment]
Hi @kilfoyle  this resulted in conflicts because it was not merged in time.  I believe I resolved the conflicts. Can you take a look?

**Comment by kilfoyle** [comment]
Hi @rseldner. Looks good! I just applied my suggestions so that we use the `{beats}` variable rather than the literal "beats". Assuming the CI checks pass, I'll merge this today. Thanks for adding this docs fix!

**Comment by kilfoyle** [comment]
By the way, I should explain this a bit better:

> As for backporting, our general policy is to backport only into active releases (the ones shown in the docs release dropdown).

We avoid pushing docs changes into releases that are no longer active because it can create an expectation that all changes are handled this way (and often we're unable to push changes into certain releases). Rather than create inconsistency, we try to stick to a policy of updating only the "active" branches. Just wanted to add some clarity about that. :-)

**Pull request merged**